packages/sdk/src/components/expand-record/ExpandRecordHeader.tsx

```diff
@@ -83,28 +94,47 @@ export const ExpandRecordHeader = (props: IExpandRecordHeader) => {
         </h4>
       )}
       {showOperator && (
-        <div>
+        <div className="flex items-center">
           <TooltipWrap description={t('expandRecord.copyRecordUrl')}>
             <Button variant={'ghost'} size={'xs'} onClick={onCopyUrl}>
               <Link />
             </Button>
           </TooltipWrap>
           {editable && (
-            <TooltipWrap
-              description={
-                recordHistoryVisible
-                  ? t('expandRecord.recordHistory.hiddenRecordHistory')
-                  : t('expandRecord.recordHistory.showRecordHistory')
-              }
-            >
-              <Button
-                variant={recordHistoryVisible ? 'secondary' : 'ghost'}
-                size={'xs'}
-                onClick={onRecordHistoryToggle}
+            <>
+              <TooltipWrap
+                description={
+                  recordHistoryVisible
+                    ? t('expandRecord.recordHistory.hiddenRecordHistory')
+                    : t('expandRecord.recordHistory.showRecordHistory')
+                }
               >
-                <History />
-              </Button>
-            </TooltipWrap>
+                <Button
+                  variant={recordHistoryVisible ? 'secondary' : 'ghost'}
+                  size={'xs'}
+                  onClick={onRecordHistoryToggle}
+                >
+                  <History />
+                </Button>
+              </TooltipWrap>
+
+              <DropdownMenu>
+                <DropdownMenuTrigger className="px-2">
+                  <Ellipsis />
+                </DropdownMenuTrigger>
+                <DropdownMenuContent>
+                  <DropdownMenuItem
+                    className="aria-selected: flex cursor-pointer items-center gap-2 px-4 py-2 text-sm text-red-500 outline-none hover:text-red-500 focus:text-red-500 aria-selected:text-red-500"
+                    onClick={async () => {
+                      await onDelete?.();
+                      onClose?.();
+                    }}
+                  >
+                    <Trash2 /> {actionStaticMap['record|delete'].description}
```

Comment:
To avoid confusion, I suggest adding a new i18n key under expandRecord in sdk.json specifically for deleting records.

Reply:
Would it not make more sense to have all `record|delete` to be accessed from a single plaace, since it is basically the same action (and text)?

Reply:
> Would it not make more sense to have all `record|delete` to be accessed from a single plaace, since it is basically the same action (and text)?

Although the current text for these actions is identical, their intended usage is quite different. We can’t guarantee that the action descriptions will remain the same in the future.